reversion/migrations/0003_auto_20160601_1600.py

```diff
@@ -21,6 +21,11 @@ def de_dupe_version_table(apps, schema_editor):
         # Add in the most recent id for each duplicate row.
         max_pk=models.Max("pk"),
     ).values_list("max_pk", flat=True)
+
+    # Do not do anything if we're keeping all ids anyway.
+    if keep_version_ids.count() == Version.objects.all().count():
+        return
+
```

Comment:
Looks like a very sensible optimization. Can I be picky and ask for the whitespace around the newlines to be removed to match the rest of the style? Then I'll merge right away.